ksql-engine/src/test/java/io/confluent/ksql/util/KafkaTopicClientTest.java

```diff
@@ -316,12 +317,13 @@ private DescribeClusterResult getDescribeClusterResult() {
     return describeClusterResult;
   }
 
-  private DescribeConfigsResult getDescribeConfigsResult() {
+  private DescribeConfigsResult getDescribeConfigsResult(boolean isCompacted) {
```

Comment:
Why this change? This variable doesn't seem to be used at all and really doesn't need to be changed relative to the other changes in this PR.

Reply:
Yes, it was left from previous changes. Removed it.